src/components/ButtonSettings.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+
+const isSettingsOpen = defineModel()
+
+function viewsettings() {
```

Comment:
```suggestion
function viewSettings() {
```
がいいなー
もしくは
```suggestion
function toggleSettings() {
```
だとなおいいかも